.gitignore

```diff
@@ -4,6 +4,7 @@
 __pycache__/
 *.py[cod]
 *$py.class
+.idea/
```

Comment:
There is a separate section for IDE stuff down below, if such things must be collected here.

This change seems unrelated though, so let's keep it in a separate commit / PR.